index.html

```diff
@@ -75,148 +101,318 @@
+
+    <!-- this is for special occasions -->
+    <div id="banner-wrapper">
+      <!-- Banner close button -->
+      <span class="banner-close" id="banner-close-button"
+        ><i class="bx bx-x"></i
+      ></span>
+      <div id="banner" class="banner">
+        <div>
+          <span class="banner-text"
+            >This is a fork of a project previously maintained by Adobe. We
+            might change names in the future.</span
+          >
+          <a
+            id="banner-button"
+            href="https://github.com/brackets-cont/brackets"
+            class="rounded radius button"
+            >Help us out!</a
+          >
         </div>
+      </div>
     </div>
-</div>
-<!---->
+    <!---->
 
-<div id="hero-wrapper">
-    <div id="hero" class="row">
+    <div id="hero-wrapper">
+      <div id="hero" class="row">
         <div class="large-12 columns">
-            <h1><span data-i18n="index.page.hero.description">Brackets is a modern, open source text editor that understands web design. We are continuing this legacy.</span></h1>
-            <div id="download">
-                <a id="download-plain-brackets" class="large rounded radius button" href="#">
-					<div data-i18n="[html]index.page.hero.download">Download Brackets <d id="download-brackets-version">1.14.2</d> (Last Adobe release)</div>
-				</a>
-                <div id="os-alert" class="alert-box radius" data-alert>
-                </div>
-                <p>
-                    <a id="other-downloads" class="nobr" href="https://github.com/brackets-cont/brackets/releases" data-i18n="index.page.hero.other-downloads">Other Downloads</a>
-                </p>
-                <!--
+          <h1>
+            <span data-i18n="index.page.hero.description"
+              >Modern, Powerful & Open source</span
+            >
+          </h1>
+          <div class="intro row">
+            <p data-i18n="index.page.content.blurb">
+              With focused visual tools and preprocessor support, Brackets is a
+              modern text editor that makes it easy to design in the browser.
+              It's crafted from the ground up for web designers and front-end
+              developers.
+            </p>
+          </div>
+          <div id="download">
+            <!-- Temporary link to download page -->
+            <a
+              id="download-plain-brackets"
+              class="large rounded radius"
+              href="#"
+            >
+              <button
+                data-i18n="[html]index.page.hero.download"
+                class="download-button"
+              >
+                Download v<d id="download-brackets-version">1.14.2</d>
+              </button>
+            </a>
+            <div id="os-alert" class="alert-box radius" data-alert></div>
+            <p>
+              <a
+                id="other-downloads"
+                class="nobr"
+                href="https://github.com/brackets-cont/brackets/releases"
```

Comment:
```suggestion
                href="https://github.com/adobe/brackets/releases"
```
We don’t have binaries or anything past adobe’s releases on our repo.